app/src/main/resources/db/migration/V1_3__create_tables.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE PDF (
+    id         SERIAL PRIMARY KEY,
+    uuid       CHAR(36)    NOT NULL UNIQUE,
+    created_at timestamptz NOT NULL,
+    pdf        bytea       NOT NULL
+);
+
+CREATE TABLE VEDTAK (
+    id                 SERIAL PRIMARY KEY,
+    uuid               CHAR(36)    NOT NULL UNIQUE,
+    created_at         timestamptz NOT NULL,
+    updated_at         timestamptz NOT NULL,
+    personident        VARCHAR(11) NOT NULL,
+    veilederident      VARCHAR(7)  NOT NULL,
+    fom                DATE        NOT NULL,
+    tom                DATE        NOT NULL,
+    begrunnelse        TEXT,
+    document           JSONB       NOT NULL DEFAULT '[]'::jsonb,
+    journalpost_id     VARCHAR(20),
+    pdf_id             INTEGER     NOT NULL UNIQUE REFERENCES PDF (id) ON DELETE CASCADE
```

Comment:
Burde vi ha noe annet enn `ON DELETE CASCADE`? Vi sier vel basically at et vedtak ikke kan finnes uten en tilhørende PDF. Bare føler det er kjipt å "miste" vedtaket dersom man kommer i skade for å slette en pdf. Pdf-en kan man jo genereres på nytt gitt `document` hvis noe feil skulle skje, og da er det dumt å ha slettet vedtaket 🤔

Reply:
Her synes jeg vi kan kanbruke ON DELETE RESTRICT.